acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -26,7 +26,7 @@
   end
 end
 
-confine_block(:to, {:platform => puppetdb_supported_platforms}, master) do
```

Comment:
No, confine seems to work as advertised (without overwriting the array of hosts in the beaker options hash).